cumulusci/robotframework/locator_manager.py

```diff
@@ -0,0 +1,149 @@
+from robot.libraries.BuiltIn import BuiltIn
+import functools
+from cumulusci.core.utils import dictmerge
+
+"""
+This module supports managing multiple location strategies. It
+works like this:
+
+0. Locators are stored in a global LOCATORS dictionary. The keys
+   are the locator prefixes, the values are dictionaries containing
+   the locators
+
+1. Libraries can register a dictionary of locators with a prefix
+   (eg: register_locators("npsp", {...})). These get added to
+   LOCATORS
+
+2. Open Test Brower calls selenium's add_location_strategy for each
+   registered set of locators. (Note: This cannot be called before a
```

Comment:
```suggestion
   registered set of locators. (Note: Location strategies cannot be added before a
```

to remove the ambiguous "this" which made me think this was saying you can't call Open Test Browser until a browser is open, until I figured out what you meant.